src/main.rs

```diff
@@ -1,65 +1,55 @@
 use anyhow::Result;
 use wasmtime::*;
 
-witx_bindgen_wasmtime::import!("spec/calculator.witx");
+witx_bindgen_wasmtime::import!("spec/component.witx");
 
-const WAT: &str = r#"
-    (module
-        (import "calculator" "add" (
-            func $calc_add
-            (param i32 i32)
-            (result i32)
-        ))
+#[allow(unused)]
+use component::*;
```

Comment:
can we use explicit imports here so it's easier to understand the code without an IDE?